gems/pending/util/miq_logger_processor.rb

```diff
@@ -122,7 +122,7 @@ def cleanup_get_next_line
 
 class MiqLoggerLine < String
   def parts
-    @parts ||= self.class.split_raw_line(self).freeze
+    @parts ||= self.class.send(:split_raw_line, self).freeze
```

Comment:
Overlap again.  Sorry.  I don't know that it makes sense to make this really private.  If we're calling the class method from an instance, it's a public method.  I'd prefer to just make it public by removing the private line.  cc @Fryguy